backend/src/main/resources/simpletrack.txt

```diff
@@ -0,0 +1,28 @@
+0,O,0
+1,O,1
+2,O,2
+3,O,3
+4,O,4
+5,O,5
+6,O,6
+7,O,7
+8,O,8
+9,O,9
+10,R,-1
+15,O,0
+20,R,1
+25,O,0.2
+30,R,-1
+35,O,0.4
+40,R,1
+45,O,0.6
```

Comment:
This line spawns an obstacle on 45% of the track progress, at 0.6 on the tracks x-axis. The last value may not be appropriate but can be changed to anything